Help/FAQ.rst

```diff
@@ -93,3 +93,33 @@ our master branch. Right now we use 7 of those checks:
 
 - **continuous-integration/appveyor/pr** This one does the same as the one
   above, but for Windows, both 32 and 64bits versions.
+
```

Comment:
commit message: use in body Adds instead of Add

also Added->Adds, and end paragraphs with a period.